### PR TITLE
UI Changes and Rating functionality

### PR DESCRIPTION
**Pull request by JancoDVTDev**
### UI Changes
- Added activity indicator for better user feedback
- Added segmented control for cards and Videos

- Background color of tableview cell from grey to white
- Added corners to custom view
- Added Shadow to custom view to display as a card
- Added ratings

### Ratings
The rating if the card is displayed by an emoji of all the users average. By tapping on the emoji the rating will be changed. This will be changed but as soon as the person loads the cards again it wil revert back to the average.

Ideally after a user click it must just tell the person that the rating has been submitted and keep displaying the average.

### Images

TipsAndTricksLoading

TipsAndTricksLoaded

Rating

Rating Changed

### CODE REVIEW

File: camshare/camshare/ViewControllers/TipsAndTricsViewController.swift [hunk 1]
```diff
@@ -11,56 +11,49 @@ import camPod
...
-        tableView.dataSource = self
-
-        let viewModel = TipsAndTricsViewModel()
-        viewModel.getTipsTricks(request: "tipsandtricks") { (result) in
-            self.tipsAndTricksContent = result
-            DispatchQueue.main.async {
-                self.tableView.reloadData()
-            }
-        }
+//        let viewModel = TipsAndTricsViewModel()
```
Comment (NathanDVT) [inline camshare/camshare/ViewControllers/TipsAndTricsViewController.swift:35]: Remove Commented out code

File: camshare/camshare/ViewControllers/TipsAndTricsViewController.swift [hunk 2]
```diff
@@ -11,56 +11,49 @@ import camPod
...
-        if status == 0 {
-            viewModel.changeStatus(status: "updatestatusnew",
-                                   model: tipsAndTricksContent[sender.tag]) { (model) in
-                                    self.tipsAndTricksContent[sender.tag] = model
-                                    self.tableView.reloadData()
-            }
+    @IBAction func segmentedControlValueChanged(segment: UISegmentedControl) {
+        if segment.selectedSegmentIndex == 0 {
+            // Cards
+            print("Cards")
```
Comment (NathanDVT) [inline camshare/camshare/ViewControllers/TipsAndTricsViewController.swift:47]: Print statement

File: camshare/camshare/ViewControllers/TipsAndTricsViewController.swift [hunk 3]
```diff
@@ -11,56 +11,49 @@ import camPod
...
 
-        if status == 1 {
-            viewModel.changeStatus(status: "updatestatustried",
-                                   model: tipsAndTricksContent[sender.tag]) { (model) in
-                                    self.tipsAndTricksContent[sender.tag] = model
-                                    self.tableView.reloadData()
-            }
+        if segment.selectedSegmentIndex == 1 {
+            // Videos
+            print("Videos")
```
Comment (NathanDVT) [inline camshare/camshare/ViewControllers/TipsAndTricsViewController.swift:54]: print Statement

File: camshare/camshare/ViewControllers/TipsAndTricsViewController.swift [hunk 4]
```diff
@@ -76,11 +69,168 @@ class TipsAndTricsViewController: UIViewController, UITableViewDelegate, UITable
...
+            currentCell.veryBadRatingImageView.tag = indexPath.row
+            currentCell.badRatingImageView.tag = indexPath.row
+            currentCell.okayRatingImageView.tag = indexPath.row
+            currentCell.goodRatingImageView.tag = indexPath.row
+            currentCell.veryGoodRatingImageView.tag = indexPath.row
+            currentCell.veryBadRatingImageView.isUserInteractionEnabled = true
+            currentCell.badRatingImageView.isUserInteractionEnabled = true
+            currentCell.okayRatingImageView.isUserInteractionEnabled = true
+            currentCell.goodRatingImageView.isUserInteractionEnabled = true
+            currentCell.veryGoodRatingImageView.isUserInteractionEnabled = true
```
Comment (NathanDVT) [inline camshare/camshare/ViewControllers/TipsAndTricsViewController.swift:119]: You can use an IBCollection in the currentCell subsection and add this in a for loop, I would suggest using NIBs for table view cells helps clean up your code.

**Review by NathanDVT — APPROVED** [review]
After you remove the prints you can merge it
The rest were just suggestions you can look into them or not.

File: camshare/camshare/ViewControllers/TipsAndTricsViewController.swift (hunk 4 above)
Comment (JancoDVTDev) [reply camshare/camshare/ViewControllers/TipsAndTricsViewController.swift:119]: Will check it out thanks

**Pull request merged**